adminSiteClient/SuggestedChartRevisionApproverPage.tsx

```diff
@@ -1206,9 +1210,23 @@ export class SuggestedChartRevisionApproverPage extends React.Component<{
                     </ul>
                 </div>
                 <div className="references">
-                    <h5>References to original chart</h5>
+                    <h2>References to original chart</h2>
                     {this.renderReferences()}
                 </div>
+                <div className="changes_summary">
+                    <h2>Variable changes</h2>{" "}
+                    {this.suggestedChartRevision &&
+                    this.suggestedChartRevision.changesInDataSummary ? (
+                        <div
+                            dangerouslySetInnerHTML={{
+                                __html: this.suggestedChartRevision
+                                    .changesInDataSummary,
+                            }}
+                        ></div>
```

Comment:
```suggestion
                        <div>
                            {this.suggestedChartRevision.changesInDataSummary}
                        </div>
```

Reply:
This appears not to render correctly. The thing is, `this.suggestedChartRevision.changesInDataSummary` is HTML-rich.

Reply:
ah, okay, yeah then that's correct. I just didn't expect it to be HTML.